utility/src/main/java/org/oppia/android/util/logging/firebase/BUILD.bazel

```diff
@@ -5,6 +5,20 @@ Logging utility implementations backed by Firebase.
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
+kt_android_library(
+    name = "firebase_exception_logger",
+    srcs = [
+        "FirebaseExceptionLogger.kt",
+    ],
+    visibility = [
+        "//utility/src/main/java/org/oppia/android/util/logging:__pkg__",
+    ],
+    deps = [
+        "//third_party:com_google_firebase_firebase-crashlytics",
+        "//utility/src/main/java/org/oppia/android/util/logging:exception_logger",
+    ],
+)
+
```

Comment:
@BenHenning @Sarthak2601 I have created this library to provide the prod exception logger as `deps` to `logging:debug_module`. Now, since it is used only here, is it the best way to do it? If no, can you please suggest a better approach? If yes, then should we add a TODO here and file an issue tracking to remove this if and when we will create a `DebugExceptionLogger` in the future? WDYT?

Reply:
I think adding a todo here along with an issue is enough. However, deferring to @BenHenning for final call on this.

Reply:
- No, there won't be a debug exception logger as part of this project.
- From what I have understood from the second and third comment, I think we should move the `DebugEventLogger` and `DebugLogreportingModule` in the `logging/firebase` base package alongside the prod impls. Then, we will crate a `debug_impl` and `debug_module` bazel libraries similar to the prod ones. This way we are not exposing the `FirebaseExceptionLogger` to `logging` package.

I have made the changes as stated above. PTAL. Thanks.